src/bin/tectonic.rs

```diff
@@ -19,49 +20,91 @@ use tectonic::status::{ChatterLevel, StatusBackend};
 
 use tectonic::{ctry, errmsg, tt_error, tt_error_styled, tt_note};
 
-fn inner(
-    args: ArgMatches,
-    config: PersistentConfig,
-    status: &mut TermcolorStatusBackend,
-) -> Result<()> {
+#[derive(Debug, StructOpt)]
+#[structopt(name = "Tectonic", about = "Process a (La)TeX document")]
+struct Opt {
+    /// The file to process, or "-" to process the standard input stream"
+    #[structopt(name = "INPUT")]
+    input: String,
+    /// The name of the "format" file used to initialize the TeX engine
+    #[structopt(long, short, name = "PATH", default_value = "latex")]
```

Comment:
If structopt is going to lowercase these, I think we should just go along with it — it is hard for me to see how changing this would cause problems for anyone.

Reply:
Agreed! I had started doing this and will push up those changes.